### ISSUE TITLE
Moving the active relayer

### ISSUE THREAD

**Issue by JohnCoperman** [MehVahdJukaar/Supplementaries#806]
MC v 1.19.4 fabric 0.14.23  Mod v 2.4.15-fabric (moonlight 2.4.16-fabric)

If the active relayer is moved, it does not provide an update to all attached lamps. This is as follows:

![supplem_relayer-lamps_1](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/0031ed7d-a00d-4915-a78a-d1471019c74d)
![supplem_relayer-lamps_2](https://github.com/MehVahdJukaar/Supplementaries/assets/111206434/0174c051-d303-44b9-98d5-f719fe774d76)

But I think... is vanilla, not supplementaries.

**Comment by MehVahdJukaar** [comment]
Yeah it's been on my to-do ever since I added that. Kind of forgot about it this issue
